lib/msf/core/exploit/remote/http/flask_unsign.rb

```diff
@@ -34,7 +34,7 @@ def get_signature(value)
             end
           end
 
-          class TimestampSigner < Signer
+          class URLSafeTimedSigner < URLSafeSigner
```

Comment:
Changed this name so if we need to add another signer in the future, we'll be in a better spot to do so before it's used all over the place.